.yamato/build_usd.yml

```diff
@@ -0,0 +1,65 @@
+﻿usd_versions:
+  - version: v19.11
+    dir_name: USD-19.11
+  - version: v20.08
+    dir_name: USD-20.08
+  - version: v20.11
+    dir_name: USD-20.11
+build_platforms:
+  - name: win
+    type: Unity::VM
+    image: filmtv/usd_build_win10_image:latest
+    flavor: b1.xlarge
+    setenv_cmd: call "C:\\Program Files (x86)\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Auxiliary\\Build\\vcvars64.bat"
+    stevedore_url: "%STEVEDORE_UPLOAD_TOOL_URL%"
+    stevedore_exe: "StevedoreUpload.exe"
+    install_path: package/com.unity.formats.usd/Runtime/Plugins/x86_64/Windows/**
+  - name: mac
+    type: Unity::VM::osx
+    image: filmtv/usd_build_macos_image:latest
+    flavor: m1.mac
+    setenv_cmd: echo "Environment set"
+    stevedore_url: "$STEVEDORE_UPLOAD_TOOL_URL"
+    stevedore_exe: "mono StevedoreUpload.exe"
+    install_path: package/com.unity.formats.usd/Runtime/Plugins/x86_64/MacOS/**
```

Comment:
I know also when I do an  macOS build, I usually have to update RPATHs manually, so that may be an issue here also.

Reply:
I was planning to set the rpath of the binaries to $ORIGIN as part of the CMake install step. Is that what you're doing on your side?

Reply:
oh nice, I was using XCode, so it's totally different